cluster/stone-prd-rh01/tenants/rhtap-o11y-tenant/integration_test_scenario.yaml

```diff
@@ -53,7 +53,7 @@ spec:
       name: application
   params:
     - name: POLICY_CONFIGURATION
-      value: rh-managed-rhtap-ser-tenant/rh-policy
+      value: rh-managed-rhtap-ser-tenant/rh-policy-o11y
```

Comment:
Same here. Point to `rhtap-releng-tenant/app-interface-standard`

Reply:
I changed it. See if this is what you had in mind.

Reply:
@ernesgonzalez33 hi, can you help me get this one merged? It's been open for few weeks and I have a couple of tasks depending on getting that repo properly released.